### PR TITLE
fix: keras import errors in new versions

### PR DESCRIPTION
**Pull request by jgongd**
## Ticket

MD-358

## Description

Update the import paths of some modules in tensorflow and Keras.

## Test Plan

Create three experiments and see if they complete successfully:
1. `example/computer_vision/iris_tf_keras/const.yaml`: Use <place_holder_tensorflow_image> in the experiment config. 
2. `example/computer_vision/iris_tf_keras/const.yaml`: Use <place_holder_tensorflow-hpc_image> in the experiment config. 
3. `example/computer_vision/iris_tf_keras/const.yaml`: Use the default experiment config.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: harness/determined/keras/_load.py [hunk 1]
```diff
@@ -3,17 +3,24 @@
...
-from tensorflow.python.training.tracking import tracking
+import tensorflow as tf
+from packaging import version
 
 import determined as det
 from determined import keras
 
+# In TF 2.14, tracking module was removed.
+if version.parse(tf.__version__) >= version.parse("2.14.0"):
+    from tensorflow.python.trackable.autotrackable import AutoTrackable  # noqa: I2041
```
Comment (MikhailKardash) [inline harness/determined/keras/_load.py:14]: we have a code style guideline where we don't want to use object names directly without modules.

Can you change both of these imports to something like:
`from tensorflow.python.trackable import autotrackable as tf_tracking`

Then you can reference the object as `tf_tracking.AutoTrackable`

**Review by MikhailKardash — COMMENTED** [review]
For the test plan, I think it's sufficient to test only the TensorFlow image changes based on what is changed in this PR.

I would update the test plan to test the NGC image and the 2.12 TF image.

**Review by MikhailKardash — APPROVED** [review]
LGTM
